project-portfolio.php

```diff
@@ -15,7 +15,7 @@
 	<div class="content">
 		<?php
 		$selected_page = 'portfolio';
-		require 'includes/nav.php';
+		require 'includes/nav.php'
```

Comment:
Ooops I'm not sure how I managed that, thanks for pointing out, now re-corrected!!!